docs/getting-started/linux-manual.md

```diff
@@ -140,12 +140,18 @@ $ docker run --rm -e SPLUNK_ACCESS_TOKEN=12345 -e SPLUNK_REALM=us0 \
 ### Custom Configuration
 
 When changes to the default configuration file is needed, a custom
-configuration file can specified and used instead. Use the `SPLUNK_CONFIG`
-environment variable or the `--config` command line argument to provide a
-custom configuration.
+configuration file can be specified and used instead. Use the `SPLUNK_CONFIG`
+environment variable or the `--config` command line argument to provide the 
+path to a file containing custom configuration YAML. Also, environment
+variable `SPLUNK_CONFIG_YAML` is available for specifying custom
+configuration YAML directly. This is useful in environments where access
+to the underlying filesystem is not readily available. For example, in 
+AWS Fargate it is more convenient to pull configuration YAML from a 
+parameter in the AWS Systems Manager Parameter Store.
```

Comment:
Suggested change: "When changes to the default configuration file are needed, create a custom configuration  YAML file. Use the `SPLUNK_CONFIG` environment variable or the `--config` command line argument to provide the path to this file. You can also use the `SPLUNK_CONFIG_YAML` environment variable to specify a custom configuration YAML directly.  This is useful in environments where access to the underlying file system is not readily available. For example, in AWS Fargate, it is more convenient to pull the configuration YAML file from a parameter in the AWS Systems Manager Parameter Store."

It's not exactly clear what this sentence means - "You can also use the `SPLUNK_CONFIG_YAML` environment variable to specify a custom configuration YAML directly. " To specify where a configuration YAML file is pulled from perhaps? Or, to specify a particular configuration YAML file that you want to use?